### PR TITLE
Onboarding storage for autogeneration

### PR DESCRIPTION
**Pull request by hunmix**
```
> azure-schema-generator@1.0.0 generate-single D:\code\azure-resource-manager-schemas\generator
> ts-node cmd/generatesingle "storage/resource-manager"

[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git fsck
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git checkout -q -- .
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git clean -q -fd
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git gc
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git fetch -q
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git checkout -q origin/master
Using whitelist config:
{
  "basePath": "storage/resource-manager",
  "namespace": "Microsoft.Storage"
}
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\y5bad7g10na --tag=all-api-versions --api-version=2015-05-01-preview --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[43.68 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2015-05-01-preview\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2015-05-01-preview
Resource Types (Resource Group Scope):
- storageAccounts
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\5adookefe6t --tag=all-api-versions --api-version=2018-03-01-preview --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
Terminate batch job (Y/N)? y
PS D:\code\azure-resource-manager-schemas\generator> npm run generate-single storage/resource-manager

> azure-schema-generator@1.0.0 generate-single D:\code\azure-resource-manager-schemas\generator
> ts-node cmd/generatesingle "storage/resource-manager"

[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git fsck
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git reset -q .
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git checkout -q -- .
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git clean -q -fd
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git gc
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git fetch -q
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git checkout -q origin/master
Using whitelist config:
{
  "basePath": "storage/resource-manager",
  "namespace": "Microsoft.Storage"
}
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\c5060xef76d --tag=all-api-versions --api-version=2015-05-01-preview --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[45.39 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2015-05-01-preview\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2015-05-01-preview
Resource Types (Resource Group Scope):
- storageAccounts
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\7dn1dbp0loo --tag=all-api-versions --api-version=2018-03-01-preview --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
WARNING: Found duplicate definition for variable-named type Microsoft.Storage/storageAccounts/managementPolicies. Skipping definition with path '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/managementPolicies/{managementPolicyName}'.
WARNING: Found duplicate definition for variable-named type Microsoft.Storage/storageAccounts/managementPolicies. Using definition with path '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/managementPolicies/{managementPolicyName}'.
[110.05 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2018-03-01-preview\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2018-03-01-preview
Resource Types (Resource Group Scope):
- storageAccounts
- storageAccounts/blobServices/containers
- storageAccounts/blobServices/containers/immutabilityPolicies
- storageAccounts/managementPolicies
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\xthxd5wr3x --tag=all-api-versions --api-version=2015-06-15 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[45.49 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2015-06-15\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2015-06-15
Resource Types (Resource Group Scope):
- storageAccounts
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\9y03szhmi2r --tag=all-api-versions --api-version=2016-01-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[45.37 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2016-01-01\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2016-01-01
Resource Types (Resource Group Scope):
- storageAccounts
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\xwpy7v6gsh --tag=all-api-versions --api-version=2016-05-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[45.32 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2016-05-01\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2016-05-01
Resource Types (Resource Group Scope):
- storageAccounts
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\3fbhe9h1xhr --tag=all-api-versions --api-version=2016-12-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[44.83 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2016-12-01\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2016-12-01
Resource Types (Resource Group Scope):
- storageAccounts
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\y2r2zzvsbvt --tag=all-api-versions --api-version=2017-06-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[49.47 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2017-06-01\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2017-06-01
Resource Types (Resource Group Scope):
- storageAccounts
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\evcs8vpoaj7 --tag=all-api-versions --api-version=2017-10-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[46.64 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2017-10-01\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2017-10-01
Resource Types (Resource Group Scope):
- storageAccounts
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\u49wdjncab --tag=all-api-versions --api-version=2018-02-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[106.95 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2018-02-01\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2018-02-01
Resource Types (Resource Group Scope):
- storageAccounts
- storageAccounts/blobServices/containers
- storageAccounts/blobServices/containers/immutabilityPolicies
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\51e0v47b59x --tag=all-api-versions --api-version=2018-07-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[108.44 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2018-07-01\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2018-07-01
Resource Types (Resource Group Scope):
- storageAccounts
- storageAccounts/blobServices
- storageAccounts/blobServices/containers
- storageAccounts/blobServices/containers/immutabilityPolicies
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\wc8utnu605 --tag=all-api-versions --api-version=2018-11-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[110.96 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2018-11-01\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2018-11-01
Resource Types (Resource Group Scope):
- storageAccounts
- storageAccounts/blobServices
- storageAccounts/blobServices/containers
- storageAccounts/blobServices/containers/immutabilityPolicies
- storageAccounts/managementPolicies
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\i1j1hmuspt --tag=all-api-versions --api-version=2019-04-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\storage\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[115.65 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2019-06-01\Microsoft.Storage.json
Provider Namespace: Microsoft.Storage
API Version: 2019-06-01
Resource Types (Resource Group Scope):
- storageAccounts
- storageAccounts/blobServices
- storageAccounts/blobServices/containers
- storageAccounts/blobServices/containers/immutabilityPolicies
- storageAccounts/encryptionScopes
- storageAccounts/fileServices
- storageAccounts/fileServices/shares
- storageAccounts/managementPolicies
- storageAccounts/privateEndpointConnections
================================================================================================================================
```

### CODE REVIEW

**Comment by leni-msft** [comment]
@blueww This is PR of onboarding storage schema autogeneration pipeline. Please help check

File: tests/2019-06-01/Microsoft.Storage.tests.json [hunk 1]
```diff
@@ -267,7 +267,7 @@
         "type": "Microsoft.Storage/storageAccounts/managementPolicies",
         "apiVersion": "2019-06-01",
         "id": "/subscriptions/{subscription-id}/resourceGroups/res7231/providers/Microsoft.Storage/storageAccounts/sto288/managementPolicies/default",
-        "name": "default",
+        "name": ".a/default",
```
Comment (blueww) [inline tests/2019-06-01/Microsoft.Storage.tests.json:270]: Why add ".a/", this is not aligned with real request.

File: schemas/2019-04-01/deploymentTemplate.json [hunk 2]
```diff
@@ -152,39 +152,6 @@
...
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices_containers" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices_containers_immutabilityPolicies" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_fileServices" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_fileServices_shares" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_managementPolicies" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_privateEndpointConnections" },
```
Comment (blueww) [inline schemas/2019-04-01/deploymentTemplate.json:179]: Why remove them? Will we add it to somewhere else?

File: schemas/2019-06-01/Microsoft.Storage.json [hunk 3]
```diff
@@ -99,60 +68,87 @@
...
+          ]
+        },
         "name": {
           "oneOf": [
             {
               "type": "string",
-              "enum": [
-                "default"
-              ]
+              "pattern": "^.*/default$"
```
Comment (blueww) [inline schemas/2019-06-01/Microsoft.Storage.json:144]: Why use this pattern? actually, it must be "default"

File: schemas/2019-04-01/deploymentTemplate.json (hunk 2 above)
Comment (leni-msft) [reply schemas/2019-04-01/deploymentTemplate.json:179]: you're right, new refs are added in [autogeneratedResources.json](https://github.com/Azure/azure-resource-manager-schemas/pull/973/files/91e7681c2c68648204ea20ba9a6270fb3a7c906a#diff-d4739cbf75decf81aeb58113dcc750afR3877-R3993) , and it is referenced back in this file here: https://github.com/Azure/azure-resource-manager-schemas/blob/91e7681c2c68648204ea20ba9a6270fb3a7c906a/schemas/2019-04-01/deploymentTemplate.json#L1260-L1262

File: schemas/2019-06-01/Microsoft.Storage.json (hunk 3 above)
Comment (leni-msft) [reply schemas/2019-06-01/Microsoft.Storage.json:144]: yes, looks weird. @hunmix could you check why?

File: schemas/2019-04-01/deploymentTemplate.json (hunk 2 above)
Comment (blueww) [reply schemas/2019-04-01/deploymentTemplate.json:179]: Good for me.

File: schemas/2019-06-01/Microsoft.Storage.json (hunk 3 above)
Comment (leni-msft) [reply schemas/2019-06-01/Microsoft.Storage.json:144]: @anthony-c-martin I found it's introduced in this commit: https://github.com/Azure/autorest.azureresourceschema/commit/0bf7eb0127cd41c1e43eb5251bff1f8572bb59bb Could you help clarify the purpose?

File: tests/2019-06-01/Microsoft.Storage.tests.json (hunk 1 above)
Comment (leni-msft) [reply tests/2019-06-01/Microsoft.Storage.tests.json:270]: Discussing in another comment, so resolve this one.

**Comment by leni-msft** [comment]
Replaced with #1130

**Pull request closed without merging**